app/javascript/check-stamp.js

```diff
@@ -1,21 +1,51 @@
-import Vue from 'vue'
-import CheckStamp from 'check-stamp.vue'
 import store from 'check-store.js'
+import CSRF from 'csrf'
+import 'whatwg-fetch'
 
 document.addEventListener('DOMContentLoaded', () => {
-  const checkStamp = document.getElementById('js-check-stamp')
-  if (checkStamp) {
-    const checkableId = checkStamp.getAttribute('data-checkable-id')
-    const checkableType = checkStamp.getAttribute('data-checkable-type')
-    new Vue({
-      store,
-      render: (h) =>
-        h(CheckStamp, {
-          props: {
-            checkableId: checkableId,
-            checkableType: checkableType
-          }
-        })
-    }).$mount('#js-check-stamp')
-  }
+  setChecks()
 })
+
+const setChecks = () => {
+  const checkStamp = document.getElementById('js-check-stamp')
+  if (!checkStamp) return
+
+  const checkableType = checkStamp.getAttribute('data-checkable-type')
+  const checkableId = checkStamp.getAttribute('data-checkable-id')
+  fetch(
+    `/api/checks.json/?checkable_type=${checkableType}&checkable_id=${checkableId}`,
+    {
+      method: 'GET',
+      headers: {
+        'X-Requested-With': 'XMLHttpRequest',
+        'X-CSRF-Token': CSRF.getToken()
+      },
+      credentials: 'same-origin'
+    }
+  )
+    .then((response) => {
+      return response.json()
+    })
+    .then((json) => {
+      const checkStampElement = document.querySelector('.stamp-approve')
+
+      if (!json[0]) return checkStampElement.classList.add('is-hidden')
+
+      checkStampElement.classList.remove('is-hidden')
```

Comment:
[toggle()](https://developer.mozilla.org/ja/docs/Web/API/DOMTokenList/toggle)メソッドを使うと１行で書けそうです！

```suggestion
      checkStampElement.classList.toggle('is-hidden', !json[0])
```

Reply:
第二引数の存在を知りませんでした！
toggleでコードを一行に、その後未チェックであればreturnするように変更しました。